<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEnvr</class>
 <widget class="QDialog" name="DialogEnvr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Environment</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxWind">
     <property name="title">
      <string>Wind and Turbulence</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="labelWindDir">
        <property name="text">
         <string>Wind Direction:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>spinBoxWindDir</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelWindSpeed">
        <property name="text">
         <string>Wind Speed:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>spinBoxWindSpeed</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="spinBoxWindDir">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelTurbulence">
        <property name="text">
         <string>Turbulence Intensity:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboTurbulence</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelWindShear">
        <property name="text">
         <string>Wind Shear Model:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>comboWindShear</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spinBoxWindSpeed">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="ComboUnitsAngle" name="comboWindDir"/>
      </item>
      <item row="1" column="2">
       <widget class="ComboUnitsVelocity" name="comboWindSpeed"/>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="comboWindShear">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Model 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Model 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Model 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Model 4</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QComboBox" name="comboTurbulence">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mild</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Moderate</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Severe</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Extreme</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxVisibility">
     <property name="title">
      <string>Visibility</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QSlider" name="sliderVisibility">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>250</number>
        </property>
        <property name="value">
         <number>250</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="spinBoxVisibility">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>100.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="ComboUnitsLength" name="comboVisibility"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>175</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxSeaLevel">
     <property name="title">
      <string>Sea Level Conditions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSeaLevelTemp">
        <property name="text">
         <string>Temperature:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>spinBoxSeaLevelTemp</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="spinBoxSeaLevelTemp">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelSeaLevelPress">
        <property name="text">
         <string>Pressure:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>spinBoxSeaLevelPress</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spinBoxSeaLevelPress">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="ComboUnitsTemperature" name="comboSeaLevelTemp"/>
      </item>
      <item row="1" column="2">
       <widget class="ComboUnitsPressure" name="comboSeaLevelPress"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Clouds</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButtonCloudsTypeLayer">
        <property name="text">
         <string>Layer</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupCloudsType</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioButtonCloudsTypeBlock">
        <property name="text">
         <string>Block</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupCloudsType</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QStackedWidget" name="stackedWidgetClouds">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageCloudsBlock">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxCloudsBlockCount">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelCloudsBlockBaseASL">
            <property name="text">
             <string>Base ASL:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>spinBoxCloudsBlockBaseASL</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelCloudsBlockCount">
            <property name="text">
             <string>Clouds per 100 sq km:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>doubleSpinBoxCloudsBlockCount</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4labelCloudsBlockThickness">
            <property name="text">
             <string>Thickness:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>spinBoxCloudsBlockThickness</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="ComboUnitsLength" name="comboCloudsBlockBaseASL"/>
          </item>
          <item row="2" column="2">
           <widget class="ComboUnitsLength" name="comboCloudsBlockThickness"/>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxCloudsBlockThickness">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="value">
             <double>500.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxCloudsBlockBaseASL">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
            <property name="value">
             <double>500.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageCloudsLayer">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinBoxCloudsLayerBaseASL">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="minimum">
             <double>100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelCloudsLayerBaseASL">
            <property name="text">
             <string>Base ASL:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="ComboUnitsLength" name="comboCloudsLayerBaseASL"/>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QComboBox" name="comboCloudsLayerCover">
            <item>
             <property name="text">
              <string>Clear</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Few</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Scattered</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Broken</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Overcast</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Cover:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Type:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboUnitsAngle</class>
   <extends>QComboBox</extends>
   <header>gui/ComboUnitsAngle.h</header>
  </customwidget>
  <customwidget>
   <class>ComboUnitsVelocity</class>
   <extends>QComboBox</extends>
   <header>gui/ComboUnitsVelocity.h</header>
  </customwidget>
  <customwidget>
   <class>ComboUnitsLength</class>
   <extends>QComboBox</extends>
   <header>gui/ComboUnitsLength.h</header>
  </customwidget>
  <customwidget>
   <class>ComboUnitsTemperature</class>
   <extends>QComboBox</extends>
   <header>gui/ComboUnitsTemperature.h</header>
  </customwidget>
  <customwidget>
   <class>ComboUnitsPressure</class>
   <extends>QComboBox</extends>
   <header>gui/ComboUnitsPressure.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinBoxSeaLevelTemp</tabstop>
  <tabstop>comboSeaLevelTemp</tabstop>
  <tabstop>spinBoxSeaLevelPress</tabstop>
  <tabstop>comboSeaLevelPress</tabstop>
  <tabstop>spinBoxWindDir</tabstop>
  <tabstop>comboWindDir</tabstop>
  <tabstop>spinBoxWindSpeed</tabstop>
  <tabstop>comboWindSpeed</tabstop>
  <tabstop>comboTurbulence</tabstop>
  <tabstop>comboWindShear</tabstop>
  <tabstop>sliderVisibility</tabstop>
  <tabstop>comboVisibility</tabstop>
  <tabstop>radioButtonCloudsTypeBlock</tabstop>
  <tabstop>radioButtonCloudsTypeLayer</tabstop>
  <tabstop>doubleSpinBoxCloudsBlockCount</tabstop>
  <tabstop>spinBoxCloudsBlockBaseASL</tabstop>
  <tabstop>comboCloudsBlockBaseASL</tabstop>
  <tabstop>spinBoxCloudsBlockThickness</tabstop>
  <tabstop>comboCloudsBlockThickness</tabstop>
  <tabstop>spinBoxCloudsLayerBaseASL</tabstop>
  <tabstop>comboCloudsLayerBaseASL</tabstop>
  <tabstop>comboCloudsLayerCover</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogEnvr</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>667</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogEnvr</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>667</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroupCloudsType"/>
 </buttongroups>
</ui>
